### ISSUE TITLE
upgrade and downgrade expect target to be relative revision

### ISSUE THREAD

**Issue by Lukas0907** [pallets-eco/flask-alembic#45]
The `downgrade` and `upgrade` commands expect the the target revision to always be relative. Therefore, e.g. `flask db downgrade 1730111612` would try to revert the last 1730111612 revisions instead of downgrading to the specific revision. Alembic and `flask-alembic` accept "absolute" revisions as well (cf. https://alembic.sqlalchemy.org/en/latest/tutorial.html#partial-revision-identifiers) so this was surprising to me.

Maybe this could be changed, or mentioned in the docs.

Thanks!

**Comment by davidism** [comment]
I bet this is an unexpected interaction with making `rev_id` a timestamp rather than random string. I pretty much only use relative numbers, so I made it unnecessary to enter `+` or `-`. When rev id was random, `int(rev_id)` was unlikely to succeed unless it was intentionally a number and not an id. But now there's no way to distinguish ids from relative numbers.

**Comment by Lukas0907** [comment]
Right, that makes sense. Using the upstream `rev_id()` function (basically reverting bd8b43e00c1fd0d8854f8be63ac245e2ebc52694) seems to restore the old behavior:

```python
from alembic import util as alembic_util

alembic.init_app(app)
alembic.rev_id = alembic_util.rev_id
```
